nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -403,7 +421,9 @@ let
       mkdir ./boot
       cp -p "${config.boot.kernelPackages.kernel}/${config.system.boot.loader.kernelFile}" \
         "${config.system.build.initialRamdisk}/${config.system.boot.loader.initrdFile}" ./boot/
-      touch --date=@0 ./EFI ./boot
+
+      # Rewrite dates for everything in the FS
+      find . -exec touch --date=@$DATE '{}' ';'
```

Comment:
You could just use `--date=2000-01-01` here to avoid magic numbers.

Reply:
TBF a magic number is magic when used with neither name or a description :).

But yes, much preferable this way.